### ISSUE TITLE
Invoking "copy-config --source running --target startup" is wiping out the startup config

### ISSUE THREAD

**Issue by bbergquist0930** [CESNET/Netopeer2#553]
Build and installed libssh, libyang, libnetopeer2, Netopeer server and CLI, and sysrepo from source (all development branch, and pulled with the latest).

Add in the "examples.yang" module.   Edit the configuration of this either through the sysrepocfg or netopeer2-cli and set a value for "/examples:cont/l".  This shows up in the "running" configuration with no issue.

From the net2peer-cli, connect and then issue

copy-config --source running --target startup

No errors are reported however the "startup" config is wiped out.   This can be seen by either sysrepocfg or  netopeer2-cli.

I think this issue is in this code "netconf.c" line 338 which looks like 

```C
    /* NACM checks */
    if (!config && (ds != SR_DS_STARTUP) && (sds != SR_DS_RUNNING)) {
        /* get source datastore data and filter them */
        sr_session_switch_ds(session, sds);
        rc = sr_get_data(session, "/*", 0, 0, 0, &config);
        if (rc != SR_ERR_OK) {
            goto cleanup;
        }
        printf("filtering the datastore\n");
        ncac_check_data_read_filter(&config, np_get_nc_sess_user(session));
    }
```
With the debugger attach, the "config == NULL" and "ds == SR_DS_STARTUP" and "sds == SR_DS_RUNNING"

So the "if" is not entered.  Then at line 381, we have

```C
        rc = sr_replace_config(session, NULL, config, NP2SRV_DATA_CHANGE_TIMEOUT, NP2SRV_DATA_CHANGE_WAIT);
```
This ends up replacing the datastore with "config" which is NULL.

I think the bug has something to do with not trying support the NACM  and filtering out the nodes that are not accessible.

**Comment by michalvasko** [comment]
Hi,
you are right, please it now, should work properly.

Regards,
Michal

**Comment by bbergquist0930** [comment]
Tested with the latest code and it now works correctly.  Thank you!.

I am not sure of the procedure, am I supposed to close the issue or will the developers close the issue?

**Comment by michalvasko** [comment]
Hi,
if the issue is fixed, you can close it. Once it is merged into `master`, it would be closed automatically.

Regards,
Michal
